packages/shared/src/common/titleCase.ts

```diff
@@ -0,0 +1,8 @@
+export function toTitleCase(str: string): string {
+  return str
+    .toLowerCase()
+    .split(" ")
+    .map(s => s.charAt(0).toUpperCase() + s.substring(1))
+    .join(" ")
+    .trim();
+}
```

Comment:
Can we use Lodash's `startCase`? Could also remove the same impl (toTitleCase) from packages/utils

Reply:
Done.